src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs

```diff
@@ -40,10 +42,21 @@ public abstract class QuicTestBase : IDisposable
         public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
         public static TimeSpan PassingTestTimeout => TimeSpan.FromMilliseconds(PassingTestTimeoutMilliseconds);
 
-        public QuicTestBase(ITestOutputHelper output)
+        public unsafe QuicTestBase(ITestOutputHelper output)
         {
             _output = output;
-            _output.WriteLine($"Using {MsQuicApi.MsQuicLibraryVersion}");
+            _output.WriteLine($"MsQuic {(IsSupported ? "supported" : "not supported")} and using '{MsQuicApi.MsQuicLibraryVersion}' version.");
+
+            if (IsSupported)
+            {
+                QUIC_SETTINGS settings = default(QUIC_SETTINGS);
+                settings.IsSet.MaxWorkerQueueDelayUs = 1;
+                settings.MaxWorkerQueueDelayUs = (uint)TimeSpan.FromSeconds(2.5).TotalMicroseconds;
+                if (StatusFailed(MsQuicApi.Api.ApiTable->SetParam(null, QUIC_PARAM_GLOBAL_SETTINGS, (uint)sizeof(QUIC_SETTINGS), (byte*)&settings)))
```

Comment:
Wouldn't it be better to set this once at the beginning instead of before every single test?

Reply:
Yeah I guess I can make it static.